pkg/arvo/sur/metadata-store.hoon

```diff
@@ -1,23 +1,26 @@
 |%
-+$  group-path  path
-+$  app-name  @tas
-+$  app-path  path
-+$  associated  (map [group-path app-name app-path] metadata)
++$  group-path    path
++$  app-name      @tas
++$  app-path      path
++$  resource      [=app-name =app-path]
++$  associations  (map [group-path resource] metadata)
 ::
 +$  metadata
   $:  title=@t
       description=@t
       color=@ux
+      date-created=@da
+      creator=@p
```

Comment:
Kinda hesitant to just casually throw these in, but can't make a solid case against them...

Reply:
Came from a discussion with Gavin. Easy to revise these over time, but we wanted to add a few obvious ones.